lib/flutter_health_fit.dart

```
@@ -329,8 +332,12 @@ class FlutterHealthFit {
   ///
   /// On Android this method works as expected.
   Future<bool> isAuthorized() async {
-    final status = await _channel.invokeMethod("isAuthorized");
-    return status;
+    try {
+      final status = await _channel.invokeMethod("isAuthorized");
```

1. Please catch only specific 2 errors that relate to the background and lack of the HealthKIt.
2. Other errors you should propagate up to consumer, otherwise you swallow all potential errors and return false results.
3. In the case you catch errors here, please print the errors, otherwise we will not understand how many these errors we have.